src/renderer/marketplace/banking/index.js

```diff
@@ -1,3 +0,0 @@
-export * from './offers-table';
```

Comment:
This will break storybook imports

Reply:
Should be fixed now, refactored directory names for consistency, updated stories.